tests/lib/hardware/node_base.py

```diff
@@ -77,6 +77,13 @@ def disk_detach(self, capacity):
         """
         pass
 
+    @abstractmethod
+    def get_volume_name(self, disk_name):
```

Comment:
I think this is a bit confusing. Should we call it `get_device_name` instead?

Reply:
Changed it